x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/utils.ts

```diff
@@ -140,12 +150,13 @@ export const searchAfterAndBulkIndex = async (
         service,
         logger
       );
-      size += searchAfterResult.hits.hits.length - 1;
-      logger.debug(`size adjusted: ${size}`);
+      sortIds = searchAfterResult.hits.hits[0].sort;
+      hitsSize += searchAfterResult.hits.hits.length;
+      logger.debug(`size adjusted: ${hitsSize}`);
       sortIds = searchAfterResult.hits.hits[0].sort;
       if (sortIds == null) {
-        logger.error('sortIds was empty search when running a signal rule');
-        return false;
+        logger.error('sortIds was empty on search');
```

Comment:
Do you still want this as an `error` even though you are returning `true` below to indicate a non-error?